Early proofreading draft.

kd8lvt opened the pull request:
- Fix Sharp Fang tooltip incorrectly stating teeth are made of bone
 - Fix common English mistakes (mainly its/it's and punctuation)
 - Spice up some repetitive tooltips between block sets
 - Spice up various tooltips lacking in character
 - Come up with a name for the Ravenous Claws' bleed ability
    - Feel free to replace it
 - Fix some awkward wording and clarify some tooltips

kd8lvt committed:
Early proofreading draft.

Fix Sharp Fang tooltip incorrectly stating teeth are made of bone
Fix common English mistakes (mainly its/it's and punctuation)
Spice up some repetitive tooltips between block sets
Spice up various tooltips lacking in character
Come up with a name for the Ravenous Claws' bleed ability
    Feel free to replace it
Fix some awkward wording amd clarify some tooltips

Elenterius commented on src/datagen/java/com/github/elenterius/biomancy/datagen/lang/EnglishLangProvider.java:
```diff
@@ -333,84 +333,84 @@ private void addBannerPatternTranslations() {
+		addItem(ModItems.EXOTIC_DUST, "Exotic Dust", "Exotic substance, perhaps the very essence of magic itself.\nAn uncommon byproduct obtained by decomposing magical things.");
+		addItem(ModItems.STONE_POWDER, "Lithic Powder", "A very basic crafting component, usually obtained as a byproduct from decomposing things.");
+
+		addItem(ModItems.NUTRIENTS, "Nutrients", "Very hard pellets rich in energy. They almost look like vitamin pills. Occasional byproduct from decomposing plants, but mainly obtained from digesting plants.");
```
you can delete the following sentence from the Nutrients tooltip as it is no longer true.
> "Occasional byproduct from decomposing plants, but mainly obtained from digesting plants."

Elenterius commented on src/datagen/java/com/github/elenterius/biomancy/datagen/lang/EnglishLangProvider.java:
```diff
@@ -194,10 +194,10 @@ protected void addTranslations() {
 
-		addTooltip("ability.bleed_proc", "Bleed Proc");
-		addTooltip("ability.bleed_proc.desc", "Adds one Bleed Stack (max 2)");
+		addTooltip("ability.bleed_proc", "Eviscerating Slash"); //"Bleed Proc" didn't seem like a finished label - Feel free to replace with whatever :) -Kd
```
very good choice of framing this as an ability, you can remove the comment.

Elenterius commented on src/datagen/java/com/github/elenterius/biomancy/datagen/lang/EnglishLangProvider.java:
```diff
@@ -252,7 +252,7 @@ private void addSoundTranslations() {
 
 		addSound(ModSoundEvents.CLAWS_ATTACK_STRONG, "Strong Claw Attack");
-		addSound(ModSoundEvents.CLAWS_ATTACK_BLEED_PROC, "Claw procs Bleed");
+		addSound(ModSoundEvents.CLAWS_ATTACK_BLEED_PROC, "Eviscerating Slash triggers"); //Again, feel free to replace with something else. -Kd
```
remove the comment

Elenterius reviewed (commented):
overall pretty good changes

kd8lvt committed:
Early proofreading draft.

Fix Sharp Fang tooltip incorrectly stating teeth are made of bone
Fix common English mistakes (mainly its/it's and punctuation)
Spice up some repetitive tooltips between block sets
Spice up various tooltips lacking in character
Come up with a name for the Ravenous Claws' bleed ability
    Feel free to replace it
Fix some awkward wording amd clarify some tooltips
Amendment 7/27/23: Implement review changes

kd8lvt commented:
A quick amendment to apply changes - sorry it took so long, totally forgot I had this open lol
